client/src/tokens/objectStyles.js

```diff
@@ -1,9 +1,12 @@
 const borderRadius = {
-  none: '0',
-  sm: '0.1875rem', // 3px
-  DEFAULT: '0.3125rem', // 5px
-  md: '0.625rem', // 10px
-  full: '100%',
+  'none': '0',
+  'sm': '0.1875rem', // 3px
+  'DEFAULT': '0.3125rem', // 5px
+  'md': '0.625rem', // 10px
+  'lg': '1rem', // 16px
+  'xl': '1.5rem', // 24px
+  '2xl': '2rem', // 32px
```

Comment:
I don’t see `lg` or `2xl` in use. If this is for a one-off component like this I think we’d be ok to use the values in the stylesheets, or arbitrary values in Tailwind.

Reply:
Discussed with Sage: will remove unused value, proceed as-is otherwise.